src/controller/tests/data_model/TestCommands.cpp

```diff
@@ -489,7 +492,10 @@ TEST_F(TestCommands, TestSuccessNoDataResponseWithClusterStatus)
 
     // Passing of stack variables by reference is only safe because of synchronous completion of the interaction. Otherwise, it's
     // not safe to do so.
-    auto onFailureCb = [&onFailureWasCalled](CHIP_ERROR aError) { onFailureWasCalled = true; };
+    auto onFailureCb = [&onFailureWasCalled](CHIP_ERROR aError) {
+        printf("   ----> %" CHIP_ERROR_FORMAT "\n", aError.Format());
```

Comment:
Why is this printf bit here?

More generally, why were any changes needed in this file?

Reply:
Missed it when I was debugging something else related to it.